go.mod

```diff
@@ -30,7 +30,7 @@ require (
 	github.com/hashicorp/go-retryablehttp v0.7.7
 	github.com/hashicorp/golang-lru/v2 v2.0.7
 	github.com/imdario/mergo v0.3.16
-	github.com/ipfs/boxo v0.24.0
+	github.com/ipfs/boxo v0.22.0
```

Comment:
Downgrade?

Reply:
good catch, I don't know why that happened. Let me try to tidy